### PR TITLE
docs: add addresses in `README.md`

### PR DESCRIPTION
**Pull request by julien-devatom**
# Pull Request

## Issue(s) fixed

This pull request:

- fixes #

### CODE REVIEW

**Review by MerlinEgalite — APPROVED** [review]

**Review by tomrpl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ You can also send an email to [security@morpho.xyz](mailto:security@morpho.xyz)
 
 ### Morpho-Aave-V3 (Ethereum)
 
-- Morpho Proxy:
-- Morpho Implementation:
-- PositionsManager:
+- Morpho Proxy: [0x33333aea097c193e66081e930c33020272b33333](https://etherscan.io/address/0x33333aea097c193e66081e930c33020272b33333)
+- Morpho Implementation: [0xcb5309bdda3463ece2d020c9d0564a95c5a90b53](https://etherscan.io/address/0xcb5309bdda3463ece2d020c9d0564a95c5a90b53)
+- PositionsManager: [0x4592e45e0c5DbEe94a135720cCfF2e4353dAc6De](https://etherscan.io/address/0x4592e45e0c5DbEe94a135720cCfF2e4353dAc6De)
```
Comment (Rubilmax) [inline README.md:59]: I really don't know if we should keep these in this README, as it adds the possibility that we forget to update them after next upgrade. Keeping the mutable values in a single place is better for maintainability

**Review by Rubilmax — APPROVED** [review]

**Review by Tristan22400 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (MerlinEgalite) [reply README.md:59]: > Keeping the mutable values in a single place is better for maintainability
Where should we put them?

Comment (julien-devatom) [reply README.md:59]: The documentation has to contain them

Comment (Rubilmax) [reply README.md:59]: Agreed; I was thinking about keeping them only in the Developer docs, not in this README

Comment (julien-devatom) [reply README.md:59]: I think for now we can kep them, and add to the upgrade process to update addresses into the docs

**Review by MathisGD — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Rubilmax) [reply README.md:59]: > I think for now we can keep them, and add to the upgrade process to update addresses into the docs

I'll never trust processes. But it's not a blocker, I approved this PR.

> Just to be sure, on `README.md` modifications it directly updates the developer doc

That's true, I forgot about it. I was actually talking about [integrators docs](https://integration.morpho.xyz), but it seems mutable addresses are not referenced there. So there's actually 0 issue with keeping them here, as of today.

Comment (MathisGD) [reply README.md:59]: >I'll never trust processes

Never trust processes, but trust CIs (I'm joking don't develop a CI for this)

**Pull request merged**